srcpkgs/plumeimpactor/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'plumeimpactor'
+pkgname=plumeimpactor
+version=1.0.0
+revision=1
+build_style=cargo
+hostmakedepends="pkgconf git cmake"
+makedepends="clang19-devel gtk+3-devel libpng-devel libjpeg-turbo-devel libglvnd-devel libxkbcommon-devel expat-devel tiff-devel libgit2-1.9-devel sqlite-devel"
+depends="usbmuxd"
+short_desc="Cross-platform iOS sideloading tool"
+maintainer="Worf <me@worf.win>"
+license="MIT"
+homepage="https://github.com/khcrysalis/PlumeImpactor"
+distfiles="https://github.com/khcrysalis/PlumeImpactor/archive/refs/tags/v${version}.tar.gz"
+checksum=12d2c28d250bcac7e3392ec247c41277c849231b18595c762392c97ef1066ed1
+
+pre_build() {
+	cargo install patch-crate
+	cargo patch-crate --force
+	cargo fetch
+}
```

Comment:
You shouldn't use `cargo install` in the template file, so you should perhaps try to manually patch faulty dependencies instead